rllib/examples/env/multi_agent.py

```diff
@@ -194,7 +199,7 @@ def reset(self):
         self.i = 0
         for i, a in enumerate(self.agents):
             self.last_obs[i] = a.reset()
-            self.last_rew[i] = None
+            self.last_rew[i] = 0
```

Comment:
why? 0 may be a pretty good reward to some envs.

Reply:
This environment is a dummy environment that we use for some tests, it isn't a wrapper. The min achievable reward by that environment is 0, so that is what I set it to.

Reply:
oh I see. ok, that sounds good.